### DISCUSSION TITLE
Is it possible to install OpenVAS on my ManagerSearch and Sensor Node?

### DISCUSSION THREAD

**Discussion by hkcs1147** [Security-Onion-Solutions/securityonion#13378] [Unsupported Versions] [answered]
I wanted to a vulnerability scan and was wondering if it's possible to install OpenVAS onto my Sensor Node.
Thanks!

**Comment by dougburks** [comment, marked as answer]
You may be able to make this work but we don't recommend or support it.

From https://docs.securityonion.net/en/2.4/best-practices.html#avoid-third-party-software-and-modifications:
> Security Onion is a free and open platform based on standard Linux distros, but we recommend treating it as an appliance and avoid installing third party software as this may conflict with our components and cause issues when updating.
